CHANGELOG.md

```diff
@@ -4,6 +4,81 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.9.0] - 2022-06-14
+
+### Added
+
+- Cluster scope install mode [#713](https://github.com/3scale/3scale-operator/pull/713)
+- Tenant CRD: delete tenant [#715](https://github.com/3scale/3scale-operator/pull/715)
+- APIManager CRD: reconcile apicast environment vars [#720](https://github.com/3scale/3scale-operator/pull/720)
+- Tenant CRD: store access token instead of the provider key [#725](https://github.com/3scale/3scale-operator/pull/725)
+- APIManager CRD: upgrade mysql to 8 [#682](https://github.com/3scale/3scale-operator/pull/682)
+- Product CRD: delete product [#723](https://github.com/3scale/3scale-operator/pull/723)
+- Backend CRD: delete backend [#723](https://github.com/3scale/3scale-operator/pull/723)
+- Tenant ownership to products and backends [#727](https://github.com/3scale/3scale-operator/pull/727)
```

Comment:
Well, you tell me, you are the author :)

I guess so. Something you still miss about it?

Reply:
Actually, I think there's no need for adding a note regarding cascading deletion. It's pretty much self-explanatory and since product or backend CR can't work without a tenant associated with it, I think we can leave it the way it is.